src/Entities/Rider.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+namespace Collection\Entities;
+
+readonly class Rider
+{
+    public int $id;
+    public string $name;
+    public string $image;
+    public string $team;
+    public string $nation;
+    public string $dob;
+    public int|null $giroGC;
+    public int|null $tourGC;
+    public int|null $vueltaGC;
+    public int|null $giroStages;
+    public int|null $tourStages;
+    public int|null $vueltaStages;
```

Comment:
There is actually a shortcut for making a type 'nullable' - you can do `?int` instead

This applies to the property types as well as the params of the constructor.

You don't HAVE to make this change now, just something you might find useful in the future